aea/manager/manager.py

```diff
@@ -263,8 +267,11 @@ async def _manager_loop(self) -> None:
                 agent_name = agents_run_tasks_futures[task]
                 self._agents_tasks.pop(agent_name)
                 if task.exception():
-                    for callback in self._error_callbacks:
-                        callback(agent_name, task.exception())
+                    if len(self._error_callbacks) == 0:
+                        self._print_exception_occurred_but_no_error_callback(agent_name)
```

Comment:
why not print the exception too? I.e. pass task.exception() here?

Reply:
Then it means we want a default callback, not a warning message

Reply:
Yeah I guess, it would be better for development